types/storybook-readme/storybook-readme-tests.tsx

```diff
@@ -0,0 +1,43 @@
+// somehow It can't find @storybook/react module, thus I couldn't write tests with storiesOf.
```

Comment:
For that you'll need to add a path mapping to "compilerOptions" in `tsconfig.json`:
```json
"paths": {
    "@storybook/react": [
        "storybook__react"
    ]
},
```

Reply:
hm... I did it before and It didn't solve the issue(because at the start, I started copying already existed storybook-addons)  
Now It solved this problem but after a few times of close/open of vscode. I don't know if it is the issue of vscode extensions(tslint?) or some reason.